code/API_definitions/qod-api.yaml

```diff
@@ -222,7 +222,67 @@ paths:
         - oAuth2ClientCredentials: []
         - threeLegged:
             - "qod-sessions-write"
-
+    get:
+      tags:
+        - QoS sessions
+      summary: Get all active sessions information
+      description: Get information about all active sessions that were created by the client
+      operationId: getSessions
+      parameters:
```

Comment:
It could be interesting to add additional filters, specially to allow filtering sessions for some specific device